src/olympia/hero/models.py

```diff
@@ -115,15 +123,15 @@ def __str__(self):
     @property
     def thumbnail_path(self):
         (path, fn) = os.path.split(self.custom_image.path)
-        return path + b'/thumbs/' + fn
+        return path + '/thumbs/' + fn
 
     @property
     def image_url(self):
         return f'{self.custom_image.url}?modified={int(self.modified.timestamp())}'
 
     @property
     def preview_url(self):
-        fn = os.path.basename(self.custom_image.path).decode('utf-8')
```

Comment:
Why did we remove the `decode`?

Reply:
`.path` was a bytestring previously, so needed to be decoded into a unicode string so it could be used in the `f` string.  Now all our paths are unicode strings.